### PR TITLE
Update the formatting and style of OWC #1

### PR DESCRIPTION
**Pull request by TPGPL**
### Description

Updated the formatting and style of OWC #1

### Status
Review

### CODE REVIEW

File: wiki/Tournaments/OWC/1/en.md [hunk 1]
```diff
@@ -1,152 +1,165 @@
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month supporter tag |
+| *Special* | 1 month supporter tag (won by ![][flag_JP] Team Japan) |
+
+## Organization
+
+The osu! World Cup #1 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+- ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993)
+- ![][flag_ES] Beuchi-chan
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: Beuchi's profile link should be added https://osu.ppy.sh/users/67192

File: wiki/Tournaments/OWC/1/en.md [hunk 2]
```diff
@@ -1,152 +1,165 @@
...
+- [Match recaps](https://osu.ppy.sh/community/forums/topics/45412?n=803)
+- [Finals recap](https://www.mediafire.com/file/ab6j6k4ihtp25o2) - provided by [dvorak](https://osu.ppy.sh/users/271359)
+
+------------------------------------------------------------
+
+## Participants
+
+### Confirmed Rosters
+
+| Country | Team Members |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: Its better to do an extra column at the start just for the flags, then center and bold the country names.
For the profile links, i think we can do that work later, many users changed their username or they are restricted.

File: wiki/Tournaments/OWC/1/en.md [hunk 3]
```diff
@@ -1,152 +1,165 @@
...
+
+![OWC #1 brackets](brackets.png "Brackets")
+
+----------------------------------------------------
+
+## Mappools
+
+### Finals
+
+| Order | Beatmap |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: Should be better to use a numbered list? also i dont know if breaks are needed.

File: wiki/Tournaments/OWC/1/en.md [hunk 4]
```diff
@@ -1,152 +1,165 @@
...
+| 2 | [Suzaku VS Genbu - Himiko (Another)](https://osu.ppy.sh/s/12710) |
+| 3 | [Susumu Hirasawa - Big Brother (KIRBY Mix)](https://osu.ppy.sh/s/10714) |
+| 4 | [NEWS - weeeek (Insane)](https://osu.ppy.sh/b/22422) |
+| 5 | [Tachibana Miya - Miya to Tengoku to Jigoku (Hentai)](https://osu.ppy.sh/s/17450) |
+| 6 | [Nobuo Uematsu - The Fierce Battle (Omega)](https://osu.ppy.sh/b/27466) |
+| Tiebreaker (3-3) | [Rhapsody - Emerald Sword (Extreme)](https://osu.ppy.sh/b/25580) |
+
+### Quarterfinals
+
+- NoMod
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: All maps should have the creator of said map and link to their respective diff (/b/) link

File: wiki/Tournaments/OWC/1/en.md [hunk 5]
```diff
@@ -1,152 +1,165 @@
...
-
-### \#1st week
-
-
-- Pick one beatmap
+In every World Cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Place | Prize(s) |
+| ----- | -------- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | 6 months supporter tag, special profile badge (![OWC 1 Profile Badge](badge.png)) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: I dont know if the profile badge is needed here.

File: wiki/Tournaments/OWC/1/en.md [hunk 6]
```diff
@@ -1,152 +1,165 @@
...
-| Semifinal, 3rd place | #7 beatmap (3-3)             |
-| Final                | #9 beatmap (4-4)             |
-
-### \#1st week
-
-
-- Pick one beatmap
+In every World Cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Place | Prize(s) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/1/en.md]: Would be better to center the "Place" column

**Comment by MegaApplePi** [comment]
Was there a reason to remove `./logo.png` and `./dedicate.png`?

**Pull request merged**